robosuite/environments/panda.py

```diff
@@ -91,12 +97,38 @@ def __init__(
             camera_depth=camera_depth,
         )
 
+    def _load_controller(self, controller_config):
+        """
+        Loads controller to be used for dynamic trajectories
+
+        @controller_config (dict): Dict of relevant controller parameters, including controller type
+            NOTE: Type must be one of: {JOINT_IMP, JOINT_TORQUE, JOINT_VEL, EE_POS, EE_POS_ORI}
+        """
+        # Add to the controller dict additional relevant params:
+        #   the robot name, mujoco sim, robot_id, joint_indexes, timestep (model) freq,
+        #   policy (control) freq, and ndim (# joints)
+        controller_config["robot_name"] = self.mujoco_robot.name
+        controller_config["sim"] = self.sim
+        controller_config["robot_id"] = "right_hand"
```

Comment:
is this supposed to correspond to the name of the robot end effector? If so, can we rename this to something else? I feel like `robot_id` is a little confusing and vague, maybe something like `robot_eef_id`?

Reply:
I agree -- switched it to "eef_name" to be more precise